test/runtests.jl

```diff
@@ -199,3 +201,23 @@ end
     @test_throws DimensionMismatch fit(KaplanMeier, [1, 2], [true])
     @test_throws ArgumentError fit(KaplanMeier, Float64[], Bool[])
 end
+
+@testset "Cox" begin
```

Comment:
I think this deserves more tests, with a few variants of the model covering all common scenarios and corner cases.